docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '3.8'
+
+services:
+  # PostgreSQL Database
+  db:
+    image: postgres:13
+    environment:
+      POSTGRES_USER: your_postgres_user
+      POSTGRES_PASSWORD: your_postgres_password
+      POSTGRES_DB: surfsense_db
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    networks:
+      - surfsense-network
+
+  # Backend Service (FastAPI)
+  backend:
+    build:
+      context: ./backend
+    ports:
+      - "8000:8000"
+    env_file:
+      - ./backend/.env
+    depends_on:
+      - db
+    networks:
```

Comment:
when backend is pointing to the hostmaschine with localhost or 127.0.0.1 you need to add `priviliged: true` else the container points at itself